lib/mastodon/domains_cli.rb

```diff
@@ -157,6 +157,48 @@ def crawl(start = nil)
       end
     end
 
+    option :url, type: :string, default: 'https://csv.rathersafe.space/f.php?d=1&h=39IM2DCd', aliases: [:u]
```

Comment:
Let's not set a default here.

Reply:
Sure @weex, will add a required url param in the command so that whenever we run it, url has to be provided.